src/filter/index.js

```diff
@@ -17,10 +17,79 @@ export default async function filter(content, selector, filterNames, filterFunct
     });
   }
 
-  const selectedContent = Array.from(webPageDOM.querySelectorAll(selector));
-  if (!selectedContent.length) {
-    console.warn(`The provided selector "${selector}" has no match in the web page.`);
+  let selectedContent = webPageDOM;
+
+  if (removeElements) {
+    let elementsToRemove = removeElements;
+
+    if (!Array.isArray(elementsToRemove)) {
```

Comment:
could be a ternary with the line above

Reply:
Or use `[].concat(elementsToRemove)` to ensure the value is an array in all cases  🙂